contracts/test/IntegrationTestBase.sol

```diff
@@ -152,6 +152,50 @@ contract TestSubnetActor is Test, TestParams {
         saLouperSelectors = SelectorLibrary.resolveSelectors("DiamondLoupeFacet");
     }
 
+    function subnetActorWithParams(
+        address gw,
+        SubnetID memory parentID
+    ) internal pure returns (SubnetActorDiamond.ConstructorParams memory) {
+        SupplySource memory native = SupplySourceHelper.native();
+        SubnetActorDiamond.ConstructorParams memory params = SubnetActorDiamond.ConstructorParams({
+            parentId: parentID,
+            ipcGatewayAddr: gw,
+            consensus: ConsensusType.Fendermint,
+            minActivationCollateral: DEFAULT_COLLATERAL_AMOUNT,
+            minValidators: DEFAULT_MIN_VALIDATORS,
+            bottomUpCheckPeriod: DEFAULT_CHECKPOINT_PERIOD,
+            majorityPercentage: DEFAULT_MAJORITY_PERCENTAGE,
+            activeValidatorsLimit: DEFAULT_ACTIVE_VALIDATORS_LIMIT,
+            powerScale: DEFAULT_POWER_SCALE,
+            permissionMode: PermissionMode.Collateral,
+            supplySource: native
+        });
+
+        return params;
+    }
+
+    function subnetActorWithParams(
+        address gw,
+        SubnetID memory parentID,
+        address tokenAddress
+    ) internal pure returns (SubnetActorDiamond.ConstructorParams memory) {
+        SubnetActorDiamond.ConstructorParams memory params = SubnetActorDiamond.ConstructorParams({
+            parentId: parentID,
+            ipcGatewayAddr: gw,
+            consensus: ConsensusType.Fendermint,
+            minActivationCollateral: DEFAULT_COLLATERAL_AMOUNT,
+            minValidators: DEFAULT_MIN_VALIDATORS,
+            bottomUpCheckPeriod: DEFAULT_CHECKPOINT_PERIOD,
+            majorityPercentage: DEFAULT_MAJORITY_PERCENTAGE,
+            activeValidatorsLimit: DEFAULT_ACTIVE_VALIDATORS_LIMIT,
+            powerScale: DEFAULT_POWER_SCALE,
+            permissionMode: PermissionMode.Collateral,
+            supplySource: SupplySource({kind: SupplyKind.ERC20, tokenAddress: tokenAddress})
+        });
+
+        return params;
+    }
```

Comment:
Can't we consolidate these two funcitons and just take a SupplySource as a parameter? The caller can use `SupplySourceHelper.native()` for a shortcut.